apps/frontend/src/index.tsx

```diff
@@ -14,6 +14,8 @@ import { Auth } from "~/pages/auth/auth.jsx";
 import { NotFound } from "~/pages/not-found/not-found.jsx";
 import { Quiz } from "~/pages/quiz/quiz.jsx";
 
+import { MyWheel } from "./pages/my-wheel/my-wheel.js";
```

Comment:
```suggestion
import { MyWheel } from "~/pages/my-wheel/my-wheel.js";
```
To have consistency with earlier imports